code/index.html

```diff
@@ -4,19 +4,94 @@
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <header class="header">
+      <img class="logo" src="./logoblue.png" alt="Dish" />
+      <img class="site-name" src="./redname.png" alt="Name" />
+
+      <!-- <span><h1>very important news</h1></span> -->
+
+      <nav class="navigation">
+        <a href="#">Home</a>
+        <a href="#">Big News</a>
+        <a href="#">Small News</a>
+        <a href="#">Contact</a>
+      </nav>
+      
     </header>
 
     <section class="big-news">
```

Comment:
Maybe this should be named differently? because it is more of a wrapper for the whole news not only the big news. It is closed only at end of the HTML code on line 94